catalog/transformer/catalog-transformer-xml/src/main/resources/OSGI-INF/blueprint/blueprint.xml

```diff
@@ -53,6 +53,7 @@
             <!-- deprecated -->
             <entry key="shortname" value="xml"/>
             <entry key="schema" value="urn:catalog:metacard"/>
+            <entry key="displayName" value="OGC XML"/>
```

Comment:
❗️ this isn't OGC XML.  Thinking this should just be called Metacard XML or just XML

Reply:
Good catch Jason.  I meant to type something more like `OGC GML` or just `GML`.  But I could also see `Metacard GML`. Sorry for the confusion @bellcc.

Reply:
No worries. This display name has been changed to `OGC GML` with this commit. 

https://github.com/codice/ddf/pull/4252/commits/506508d955d0c64eda2f89d7e159fe258c3081a7